CHANGELOG.md

```diff
@@ -3,13 +3,15 @@
 This changelog keeps track of all changes to the Packs SDK. We follow conventions from [keepachangelog](https://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
+### Added
```

Comment:
Nit, newline above. Mostly doesn't matter just paranoid if perhaps the tooling that creates these releases chokes on expected formatting.